test/calculatorGtest.cpp

```diff
@@ -15,34 +15,30 @@ using namespace calculator;
 
 TEST(WeekOneCalculatorTests, CalculateMethodTests)
```

Comment:
Couple notes for tests

- The test name should be roughly When[FunctionName]CalledOn[ObjectName]with[Parameters]_Then[SomethingHappens]And[Object]Returned
- Yes it is very verbose, but if a section isn't relevant you don't have to include it, here is an example from mine.
```
TEST(DivideCalculatorTests, WhenToStringIsCalledOnDivideCalculators_ThenFullFormattedEquationIsReturned)
{...}
```
- Individual tests should only test one thing. i.e., you should refactor to something like this
```
TEST(WeekOneCalculatorTests, WhenCalculateIsCalledWithValidOperation_ThenResultOutParamIsSet)
{
//...
Calculate('+', 1, 1, result)
EXPECT_TRUE(result == 2);
//...
}

TEST(WeekOneCalculatorTests, WhenCalculateIsCalledWithValidOperation_ThenNullptrIsReturned)
{
//...
   EXPECT_EQ( Calculate('*', 1, -1, result), nullptr);
//...
}
```